### PR TITLE
pulumi 2.9.0

### PR DESCRIPTION
**Pull request by stack72**
Bumped with brew bump-formula-pr

### CODE REVIEW

**Comment by SMillerDev** [comment]
Please use the commit style `pulumi 2.9.0`

**Comment by stack72** [comment]
@SMillerDev done

File: Formula/pulumi.rb [hunk 1]
```diff
@@ -17,28 +17,21 @@ class Pulumi < Formula
...
-      cd "./sdk" do
-        system "go", "mod", "download"
-      end
-      cd "./pkg" do
-        system "go", "mod", "download"
-      end
-      system "make", "brew"
-      bin.install Dir["#{buildpath}/bin/*"]
-      prefix.install_metafiles
+    bin.install Dir["#{ENV["GOPATH"]}/bin/pulumi*"]
```
Comment (SMillerDev) [inline Formula/pulumi.rb:29]: This shouldn't work right?

Comment (stack72) [reply Formula/pulumi.rb:29]: I didn't add this - this wasn't in my original PR - this was a force push by @dawidd6

My original PR had 2 values change and nothing else!

Comment (dawidd6) [reply Formula/pulumi.rb:29]: Why not?

Comment (SMillerDev) [reply Formula/pulumi.rb:29]: Cause GOPATH isn't set anywhere?

Comment (dawidd6) [reply Formula/pulumi.rb:29]: It is:
https://github.com/Homebrew/brew/blob/778bd543c72df0fbee3e7562a3bc89fc6d8b7f32/Library/Homebrew/formula.rb#L2142

**Comment by stack72** [comment]
Hi both

I really wish this PR wasn’t force updated with this new code - it’s been greater than 24 hours now since I opened this and the only feedback I needed to address was the commit message

Please can this be resolve so that we can allow our users to be able to be upgrade when they use homebrew?

Thanks

Paul

**Comment by dawidd6** [comment]
The formula builds fine, I don't see any problems with it.

**Review by dawidd6 — APPROVED** [review]

**Pull request closed without merging**